doc/src/site/sphinx/deployment/backends.rst

```diff
@@ -164,9 +164,9 @@ To start an H2O cluster and connect to it, run:
         In case we stored the path of the driver H2O jar to environmental variable ``H2O_DRIVER_JAR``, we don't
         have to specify ``setH2ODriverPath`` as Sparkling Water will read the path from the environmental variable.
 
-When specifying the queue, we recommend that this queue has YARN preemption off in order to have stable a H2O cluster.
+When specifying the queue, we recommend that this queue has YARN preemption off in order to have stable an H2O cluster.
 
-In case of Scala, it can also happen that we might need to explicitly set the client's IP or network. To see how this can be configured, please
+In the case of Scala, it can also happen that we might need to explicitly set the client's IP or network. To see how this can be configured, please
```

Comment:
why `the`?

Reply:
Seems good to me -> see https://englishstudypage.com/grammar/in-case-of-vs-in-the-case-of-in-english/

Reply:
ok, thanks!